modules/document_repository/README.md

```diff
@@ -0,0 +1,41 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides a centralized location for non-candidate documents that are relevant to the study (i.e. unfilled versions of instruments, publications, or manual of operations).
+
+## Intended Users
+
+ Given the appropriate permissions, LORIS users can view, download, upload, and delete files, as well as edit information pertaining to those files.
+ This module is used primarily by study coordinators/administrators who share study protocols and manuals, as well as data entry staff who consult these documents.
```

Comment:
I would not limit access to the doc repository to data entry staff. People doing analyses might need to have access to the doc repo to have access to publications and be able to see what the instruments look like or the project documentation. I think the doc repository should be accessible to all users as it contains pertinent information regarding the project description.